distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.15 (in development)
 ------------------------------------------------------------------------
+- Fix: [#22653] Add several .parkpatch files for missing water tiles in RCT1 and RCT2 scenarios.
```

Comment:
These entries are incorrect in all parkpatch PRs. It should list the _problem_, and _not_ the fix.